packages/xrpl/test/integration/requests/subscribe.test.ts

```diff
@@ -156,16 +156,7 @@ describe('subscribe', function () {
         const ledgerResponse: LedgerStreamResponse =
           response.result as LedgerStreamResponse
         // Explicitly checking that there are only known fields in the return
-        const expectedResult = {
-          fee_base: ledgerResponse.fee_base,
-          fee_ref: ledgerResponse.fee_ref,
-          ledger_hash: ledgerResponse.ledger_hash,
-          ledger_index: ledgerResponse.ledger_index,
-          ledger_time: ledgerResponse.ledger_time,
-          reserve_base: ledgerResponse.reserve_base,
-          reserve_inc: ledgerResponse.reserve_inc,
-          validated_ledgers: ledgerResponse.validated_ledgers,
-        }
+        const expectedResult = { ...ledgerResponse }
```

Comment:
I think this defeats the point of this test - the reason those fields were explicitly defined was that we were trying to ensure that if a new field was added, or one was removed, our test would fail. (As it is now, the assertion is irrelevant because we're dynamically updating our expected value to have the same shape no matter what)

Reply:
I see.  In this case the `fee_ref` field is removed if the SetFee amendment is enabled.  Do you suggest I remove that one field?

Reply:
I think removing that one field makes sense to me :)

Reply:
Done. PR description updated.